### PR TITLE
Added entries for places where datafiles generated by simulations or data analyses are often stored temporarily

### PR DESCRIPTION
**Pull request by apdavison**

### CODE REVIEW

File: instances/fileRepositoryType/seafile.jsonld [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "@id": "https://openminds.ebrains.eu/instances/fileRepositoryType/seafile",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/FileRepositoryType",
+    "name": "Seafile repository",
+    "definition": "Seafile is an open source file sync&share solution designed for high reliability, performance and productivity.",
```
Comment (lzehl) [inline instances/fileRepositoryType/seafile.jsonld:5]: Is "sync&share" on purpose without spacing?

**Review by lzehl — COMMENTED** [review]
If the spacing is correct (see comment) let me know. I'll merge the PR then.

File: instances/fileRepositoryType/seafile.jsonld (hunk 1 above)
Comment (apdavison) [reply instances/fileRepositoryType/seafile.jsonld:5]: this was copied verbatim from the Seafile website

Comment (lzehl) [reply instances/fileRepositoryType/seafile.jsonld:5]: Then we leave it like this.

**Review by lzehl — APPROVED** [review]

**Pull request merged**